antsibull/docs_parsing/routing.py

```diff
@@ -206,11 +220,23 @@ async def load_collection_routing(collection_name: str,
     if collection_name == 'ansible.builtin':
         # ansible-base/-core has a special directory structure we currently do not want
         # (or need) to handle
-        return
+        return plugin_routing_out
+
+    for plugin_type in DOCUMENTABLE_PLUGINS:
+        directory_name = 'modules' if plugin_type == 'module' else plugin_type
+        directory_path = os.path.join(collection_metadata.path, 'plugins', directory_name)
+        plugin_type_routing = plugin_routing_out[plugin_type]
+
+        symlink_redirects = find_symlink_redirects(collection_name, plugin_type, directory_path)
+        for redirect_name, redirect_dst in symlink_redirects.items():
+            if 'redirect' not in plugin_type_routing[redirect_name]:
```

Comment:
This will raise an `AttributeError` if `redirect_name not in plugin_type_routing`. That actually does happen for community.general. Adding
```.py
            if redirect_name not in plugin_type_routing:
                plugin_type_routing[redirect_name] = {}
```
before this line makes it work. Or at least a bit longer, until the code crashes in `find_flatmapping_short_long_maps` because `redirect = routing_data['redirect']` is not a string, but a dictionary with a key `redirect` :)

Reply:
Fixed in bac0aca263ba6aeb8074623e5cff17356b41befb.